pkg/arm/data/master-startup.sh

```diff
@@ -651,6 +651,36 @@ sed -i -re "s#( *server: ).*#\1https://$(hostname)#" /etc/origin/node/node.kubec
 # daemonset will not fail and set the master node labels correctly.
 cp /etc/origin/node/node.kubeconfig /etc/origin/node/bootstrap.kubeconfig
 
+{{- if $.Extra.IsRecovery }}
+logger -t master-startup.sh "starting recovery on $(hostname)"
+# step 1 get the backup
+tempBackDir=$(mktemp -d -t etcdbackup-XXXXXX)
```

Comment:
I'm worried that disk space is going to be an issue here in any non-toy example.  If you save the snapshot to the same disk, presumably it should work for dbs up to approx 1/2 of the disk size?  Presumably any bigger will actually require a temporary restore disk or something?  Suggest you implement the former and open a story for the latter.

Reply:
https://github.com/openshift/openshift-azure/issues/785